tests/cdc-bank/bank.go

```diff
@@ -345,7 +345,11 @@ func getDDLEndTs(db *sql.DB, tableName string) (result string, err error) {
 			return "", err
 		}
 		if line.JobType == "create table" && line.TblName == tableName && line.State == "synced" {
-			return line.EndTime, nil
+			if line.EndTime == nil {
+				log.Warnf("ddl end time is null, line=%+v", line)
+				return "", nil
```

Comment:
Should we continue if the endTs is empty [here](https://github.com/pingcap/tipocket/blob/c63af1c8f1f5ed86297bff9a7c0567d0565058e3/tests/cdc-bank/bank.go#L220)? Otherwise we set `@@tidb_snapshot` to an empty string, it might be unexpected.